packages/base/src/UI5Element.js

```diff
@@ -307,6 +312,10 @@ class UI5Element extends HTMLElement {
 				this._detachChildPropertyUpdated(child);
 			}
 
+			if (child.isUI5Element && slotData.invalidateParent) {
```

Comment:
I'm not sure if needed, but the "child" check is missing, like on line 311. IMO just set _shouldInvalidateParent  to false undonditionally  after line 312.